collectors/python.d.plugin/pandas/metadata.yaml

```diff
@@ -0,0 +1,310 @@
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
+      folding:
+        title: Config options
+        enabled: true
+      list:
+        - name: chart_configs
+          description: an array of chart configuration dictionaries
+          default_value: []
+          required: true
+        - name: chart_configs.name
+          description: name of the chart to be displayed in the dashboard.
+          default_value: None
+          required: true
+        - name: chart_configs.title
+          description: title of the chart to be displayed in the dashboard.
+          default_value: None
+          required: true
+        - name: chart_configs.family
+          description: "[family](https://learn.netdata.cloud/docs/data-collection/chart-dimensions-contexts-and-families#family) of the chart to be displayed in the dashboard."
+          default_value: None
+          required: true
+        - name: chart_configs.context
+          description: "[context](https://learn.netdata.cloud/docs/data-collection/chart-dimensions-contexts-and-families#context) of the chart to be displayed in the dashboard."
+          default_value: None
+          required: true
+        - name: chart_configs.type
+          description: the type of the chart to be displayed in the dashboard.
+          default_value: None
+          required: true
+        - name: chart_configs.units
+          description: the units of the chart to be displayed in the dashboard.
+          default_value: None
+          required: true
+        - name: chart_configs.df_steps
+          description: a series of pandas operations (one per line) that each returns a dataframe.
+          default_value: None
+          required: true
+        - name: update_every
+          description: Sets the default data collection frequency.
+          default_value: 5
+          required: false
+        - name: priority
+          description: Controls the order of charts at the netdata dashboard.
+          default_value: 60000
+          required: false
+        - name: autodetection_retry
+          description: Sets the job re-check interval in seconds.
+          default_value: 0
+          required: false
+        - name: penalty
+          description: Indicates whether to apply penalty to update_every in case of failures.
+          default_value: yes
+          required: false
+        - name: name
+          description: Job name. This value will overwrite the `job_name` value. JOBS with the same name are mutually exclusive. Only one of them will be allowed running at any time. This allows autodetection to try several alternatives and pick the one that works.
+          default_value: ""
+          required: false
+    examples:
+      folding:
+        enabled: true
+        title: Config
+      list:
+        - name: Temperature API Example
+          folding:
```

Comment:
nice, you folded them, but no need to mention in each specifically, you have the global folding set for that reason! No need to change though